src/main/webapp/deadlineExceededErrorPage.jsp

```diff
@@ -9,7 +9,7 @@
     <title>TEAMMATES</title>
     <link rel="stylesheet" href="/bootstrap/css/bootstrap.min.css" type="text/css"/>
     <link rel="stylesheet" href="/bootstrap/css/bootstrap-theme.min.css" type="text/css"/>
-    <link rel="stylesheet" href="stylesheets/teammatesCommon.css" type="text/css">
+    <link rel="stylesheet" href="/stylesheets/teammatesCommon.css" type="text/css">
```

Comment:
It was during the refactoring. Somehow the piece of code that I was refactoring thrown NullPointerException back then. So when I took a look at the server side console, I discovered it.

Ok. I will leave this out from the patch.